spec/forms/admission_webhook_form_spec.rb

```diff
@@ -0,0 +1,105 @@
+require "rails_helper"
+
+describe AdmissionWebhookForm do
+  describe "validations" do
+    let(:cert_file) do
+      Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", "admin.crt"))
```

Comment:
I think this code can just use `fixture_file_upload` instead of `Rack::Test::UploadedFile`.